test/sharness/lib/test-lib.sh

```diff
@@ -26,6 +26,7 @@ fi
 # it's too late to pass in --verbose, and --verbose is harder
 # to pass through in some cases.
 test "$TEST_VERBOSE" = 1 && verbose=t
+test "$TEST_IMMEDIATE" = 1 && immediate=t
```

Comment:
?

Reply:
Oh yeah, if you want we can remove this, but in that case I'd open a new PR for this.

When running single tests, I usually run them with `./txxxx-foo-bar.sh -v -i`, which results in the test aborting immediately after an error. This makes finding the failing test in a large test file a lot easier. For some reason this behaviour could not be set using the env, meaning that it could not be used when invoked with `make`. This line allows enabling this mode using `TEST_IMMEDIATE=1 make`.

Reply:
Got it, sounds useful, let's keep it (although we should probably document it in the sharness readme).